firebase_admin/_auth_client.py

```diff
@@ -720,7 +724,9 @@ def list_saml_provider_configs(
         """
         return self._provider_manager.list_saml_provider_configs(page_token, max_results)
 
-    def _check_jwt_revoked(self, verified_claims, exc_type, label):
+    def _check_jwt_revoked_or_disabled(self, verified_claims, exc_type, label):
         user = self.get_user(verified_claims.get('uid'))
         if verified_claims.get('iat') * 1000 < user.tokens_valid_after_timestamp:
             raise exc_type('The Firebase {0} has been revoked.'.format(label))
+        if user.disabled:
```

Comment:
Should `user.disabled` takes precedent over revoked token here? If the user is disabled, do we need to check the token at all? :)